widgetssdk/src/main/java/com/glia/widgets/callvisualizer/ActivityWatcherForCallVisualizerController.kt

```diff
@@ -167,6 +169,12 @@ internal class ActivityWatcherForCallVisualizerController(
     private fun addScreenSharingCallback(activity: Activity) {
         // Call and Chat screens process screen sharing requests on their own.
         if (callVisualizerController.isCallOrChatScreenActiveUseCase(activity)) return
+        if (!isCallVisualizerUseCase.invoke()) {
```

Comment:
This can be shortened to `isCallVisualizerUseCase()`

Reply:
Yes, thank you!